sympy/sets/sets.py

```diff
@@ -729,9 +729,10 @@ def __new__(cls, start, end, left_open=False, right_open=False):
             raise ValueError("Non-real intervals are not supported")
 
         # evaluate if possible
+        cls.is_EmptySet = (end - start).is_negative
```

Comment:
`.is_EmptySet` should not be a class attribute since its value may be different for different objects.